### PR TITLE
Create findbugs-annotations_ubuntu_20.04.sh

### PR DESCRIPTION
**Pull request by nageshlop**
Creating build script for package specific file. Build the package at local VM and is successful.

### CODE REVIEW

**Comment by nageshlop** [comment]
BUILD LOGS:
 
Apache Maven 3.6.3
Maven home: /usr/share/maven
Java version: 11.0.10, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-ppc64el
Default locale: en, platform encoding: UTF-8
OS name: "linux", version: "5.4.0-42-generic", arch: "ppc64le", family: "unix"
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Scanning for projects...
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/scm/maven-scm-provider-gitexe/1.3/maven-scm-provider-gitexe-1.3.pom
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/scm/maven-scm-provider-gitexe/1.3/maven-scm-provider-gitexe-1.3.pom (2.3 kB at 2.0 kB/s)
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/scm/maven-scm-providers-git/1.3/maven-scm-providers-git-1.3.pom
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/scm/maven-scm-providers-git/1.3/maven-scm-providers-git-1.3.pom (2.2 kB at 16 kB/s)

kB/s)
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/commons-io/commons-io/2.2/commons-io-2.2.jar (174 kB at 1.1 MB/s)
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-archiver/2.9/plexus-archiver-2.9.jar (145 kB at 909 kB/s)
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/shared/maven-shared-utils/0.7/maven-shared-utils-0.7.jar (170 kB at 1.0 MB/s)
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.9/commons-compress-1.9.jar (378 kB at 1.1 MB/s)
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ findbugs-annotations ---
[INFO] Installing /scratch/jar-proj/findbugs-annotations/target/findbugs-annotations-3.0.1-2-SNAPSHOT.jar to /root/.m2/repository/com/github/stephenc/findbugs/findbugs-annotations/3.0.1-2-SNAPSHOT/findbugs-annotations-3.0.1-2-SNAPSHOT.jar
[INFO] Installing /scratch/jar-proj/findbugs-annotations/pom.xml to /root/.m2/repository/com/github/stephenc/findbugs/findbugs-annotations/3.0.1-2-SNAPSHOT/findbugs-annotations-3.0.1-2-SNAPSHOT.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  29.291 s
[INFO] Finished at: 2021-04-22T05:29:00Z
[INFO] ------------------------------------------------------------------------
ubuntu@nagesh-at:/scratch/jar-proj/findbugs-annotations$

File: f/findbugs-annotations/findbugs-annotations_ubuntu_20.04.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        exit 0
+else
+        echo "Cloning the target"
+fi
+
+# Build and test the package
+
+git clone --depth 1 --branch $BRANCH $URL --single-branch
+cd $PKG_NAME
+sudo mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```
Comment (gururajrkatti) [inline f/findbugs-annotations/findbugs-annotations_ubuntu_20.04.sh:51]: What is the reason we are "skipping" the tests ?

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Review by gururajrkatti — CHANGES_REQUESTED** [review]
Ok. pl. squash the commits. Hope you have updated the latest test logs in the PR

**Comment by nageshlop** [comment]
@gururajrkatti and @seth-priya  squashed the commits and verified the test results. Let me know if any changes required.

**Review by gururajrkatti — APPROVED** [review]
LGTM

**Pull request merged**